app/views/shared/_my-booking-card.html.erb

```diff
@@ -0,0 +1,18 @@
+<% if @my_bookings.empty? %>
+  <p>Wow, such empty! 👶</p>
```

Comment:
Lol! What if we add a call to action here, @zuccamia @attinore? Something like:
`<p>No bookings found.</p>
<p><%= link_to 'Create a new booking?', listings_path %></p>`

Reply:
@worksofgenius Yeah I think a call to action would work well here too! Will add this! 
Just a suggestion but what 'Find an activity' for 'Create a new booking?'? (I thought it sounds a bit more intuitive to me 🤔 )

Reply:
I like it!